_hardware/04-CompatibilityCharts/05-Misc/index.md

```diff
@@ -27,7 +27,24 @@ has_toc: false
 <h2 align="center">An info dump of obscure hardware support</h2>
 <br>
 
-<p align="center">This section is currently a placeholder, and may even be removed!</p>
+<p align="center">This section includes hardware that doesn't fall into any of the previous categories.</p>
+
+# Fingerprint Sensors
+
+- Fingerprint sensors won't work in macOS.
+
+# Windows Hello Face Recognition (FaceID)
+
+- WFHR/FaceID won't work in macOS. There is chance that you could get your camera working, but that's it.
+
+# CD-DVD ROM
+
+- Nearly any SATA optical drive will work, though you may need to use a specific SMBIOS.
+
+# Peripherals
+
+- Whatever you got, it will work.
```

Comment:
Hm - this is a little bit too lazy. Peripherals depend on macOS drivers - if theres no driver, it wont work. You should let the reader know that.

Reply:
tbf, a majority of USB periphs are supported because of the IOUSBHost stack having HID device support

though, the same can’t be said for certain pieces of hardware, like an RGB controller or something

Reply:
also let’s not forget Apple removing drivers in the case of real Macs after they’ve been removed from the OS cycle

Reply:
True, but im mainly talking about devices like audio interfaces: if there's no driver for macOS, you're screwed

Reply:
that makes sense